### DISCUSSION TITLE
Installation problem/Unable to import from TeslaFi

### DISCUSSION THREAD

**Discussion by giovaorama** [teslamate-org/teslamate#4160] [Q&A]
Hi I just installed TeslaMate following the docker installation.

I'm still using the default docker-compose.yml, because each time I tried to set a different password leads to authentication error for the user teslamate in the postgres.

Anyway, I'm trying to import the data from TeslaFi, but it fails with this error

```
storage-teslamate-1  | 2024-08-26 18:20:41.923 [info] Importing 1 file(s) ...
storage-teslamate-1  | 2024-08-26 18:20:41.941 [info] Starting logger for 'Melody Pond'
storage-teslamate-1  | 2024-08-26 18:20:41.950 [error] GenStateMachine TeslaMate.Import terminating
storage-teslamate-1  | ** (MatchError) no match of right hand side value: {:error, #Ecto.Changeset<action: :update, changes: %{eid: 45308}, errors: [vid: {"is invalid", [type: :integer, validation: :cast]}], data: #TeslaMate.Log.Car<>, valid?: false>}
storage-teslamate-1  |     (teslamate 1.30.1) lib/teslamate/vehicles.ex:113: TeslaMate.Vehicles.create_or_update!/1
storage-teslamate-1  |     (teslamate 1.30.1) lib/teslamate/import.ex:289: TeslaMate.Import.create_car/1
storage-teslamate-1  |     (teslamate 1.30.1) lib/teslamate/import.ex:128: TeslaMate.Import.handle_event/4
storage-teslamate-1  |     (stdlib 5.2.3) gen_statem.erl:1397: :gen_statem.loop_state_callback/11
storage-teslamate-1  |     (stdlib 5.2.3) proc_lib.erl:241: :proc_lib.init_p_do_apply/3

```

Anyway can help me to understand what is the problem?
I do not think i'm making something strange.

Thanks

**Comment by giovaorama** [comment]
I solve the problem with the installation. I clean all the containers and volume and restart from scratch. No more login problems

But I still cannot import TeslaFi data.

Always the same error in the logs for teslamate

**Reply by jakobfriis** [reply]
It seems to complain, that your vehicle id is invalid. Check your csv file - i had to change all vehicle id values to 1

**Reply by giovaorama** [reply]
Exactly this... the vehicle_id rows have a sort of counters, something like '75 75 0 0 0 0 0' with the first number always increasing

**Comment by giovaorama** [comment]
Just change the vehicle_id column in all the rows 1
